[{"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.bccache.Bucket.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.bccache.Bucket.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing _MemcachedClient type annotation for jinja2.bccache.MemcachedBytecodeCache.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation _MemcachedClient for jinja2.bccache.MemcachedBytecodeCache.__init__. Import of _MemcachedClient failed: name '_MemcachedClient' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.bccache.BytecodeCache.get_bucket: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.bccache.BytecodeCache.get_bucket. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.compiler.CodeGenerator.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.compiler.CodeGenerator.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.compiler.generate: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.compiler.generate. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.NoReturn type annotation for jinja2.compiler.CodeGenerator.fail: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.NoReturn for jinja2.compiler.CodeGenerator.fail. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.Literal['\\n', '\\r\\n', '\\r'] type annotation for jinja2.defaults.NEWLINE_SEQUENCE: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.Literal['\\n', '\\r\\n', '\\r'] for jinja2.defaults.NEWLINE_SEQUENCE. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.Literal['\\n', '\\r\\n', '\\r'] type annotation for jinja2.environment.Environment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.Literal['\\n', '\\r\\n', '\\r'] for jinja2.environment.Environment.__init__. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] type annotation for jinja2.environment.Environment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] for jinja2.environment.Environment.__init__. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('BaseLoader')] type annotation for jinja2.environment.Environment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('BaseLoader')] for jinja2.environment.Environment.__init__. Import of BaseLoader failed: name 'BaseLoader' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('BytecodeCache')] type annotation for jinja2.environment.Environment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('BytecodeCache')] for jinja2.environment.Environment.__init__. Import of BytecodeCache failed: name 'BytecodeCache' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Dict[str, ForwardRef('Extension')] type annotation for jinja2.environment.load_extensions: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Dict[str, ForwardRef('Extension')] for jinja2.environment.load_extensions. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] type annotation for jinja2.environment.load_extensions: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] for jinja2.environment.load_extensions. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Union[str, typing.Type[ForwardRef('Extension')]] type annotation for jinja2.environment.Environment.add_extension: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Union[str, typing.Type[ForwardRef('Extension')]] for jinja2.environment.Environment.add_extension. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.Literal['\\n', '\\r\\n', '\\r'] type annotation for jinja2.environment.Environment.overlay: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.Literal['\\n', '\\r\\n', '\\r'] for jinja2.environment.Environment.overlay. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] type annotation for jinja2.environment.Environment.overlay: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] for jinja2.environment.Environment.overlay. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('BaseLoader')] type annotation for jinja2.environment.Environment.overlay: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('BaseLoader')] for jinja2.environment.Environment.overlay. Import of BaseLoader failed: name 'BaseLoader' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('BytecodeCache')] type annotation for jinja2.environment.Environment.overlay: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('BytecodeCache')] for jinja2.environment.Environment.overlay. Import of BytecodeCache failed: name 'BytecodeCache' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Iterator[ForwardRef('Extension')] type annotation for jinja2.environment.Environment.iter_extensions: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Iterator[ForwardRef('Extension')] for jinja2.environment.Environment.iter_extensions. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.NoReturn type annotation for jinja2.environment.Environment.handle_exception: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.NoReturn for jinja2.environment.Environment.handle_exception. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Union[ForwardRef('TokenStream'), typing.Iterable[ForwardRef('Token')]] type annotation for jinja2.ext.Extension.filter_stream: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Union[ForwardRef('TokenStream'), typing.Iterable[ForwardRef('Token')]] for jinja2.ext.Extension.filter_stream. Import of TokenStream failed: name 'TokenStream' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing TokenStream type annotation for jinja2.ext.Extension.filter_stream: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation TokenStream for jinja2.ext.Extension.filter_stream. Import of TokenStream failed: name 'TokenStream' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Parser type annotation for jinja2.ext.Extension.parse: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Parser for jinja2.ext.Extension.parse. Import of Parser failed: name 'Parser' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Parser type annotation for jinja2.ext.InternationalizationExtension.parse: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Parser for jinja2.ext.InternationalizationExtension.parse. Import of Parser failed: name 'Parser' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Parser type annotation for jinja2.ext.ExprStmtExtension.parse: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Parser for jinja2.ext.ExprStmtExtension.parse. Import of Parser failed: name 'Parser' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Parser type annotation for jinja2.ext.LoopControlExtension.parse: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Parser for jinja2.ext.LoopControlExtension.parse. Import of Parser failed: name 'Parser' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Parser type annotation for jinja2.ext.DebugExtension.parse: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Parser for jinja2.ext.DebugExtension.parse. Import of Parser failed: name 'Parser' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.make_attrgetter: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.make_attrgetter. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.make_multi_attrgetter: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.make_multi_attrgetter. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing t.Union[str, HasHTML] type annotation for jinja2.filters.do_forceescape: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation t.Union[str, HasHTML] for jinja2.filters.do_forceescape. Import of HasHTML failed: name 'HasHTML' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing EvalContext type annotation for jinja2.filters.do_replace: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation EvalContext for jinja2.filters.do_replace. Import of EvalContext failed: name 'EvalContext' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing EvalContext type annotation for jinja2.filters.do_xmlattr: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation EvalContext for jinja2.filters.do_xmlattr. Import of EvalContext failed: name 'EvalContext' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.Literal[\"key\", \"value\"] type annotation for jinja2.filters.do_dictsort: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.Literal[\"key\", \"value\"] for jinja2.filters.do_dictsort. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_sort: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_sort. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_unique: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_unique. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_min: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_min. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_max: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_max. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing EvalContext type annotation for jinja2.filters.sync_do_join: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation EvalContext for jinja2.filters.sync_do_join. Import of EvalContext failed: name 'EvalContext' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing EvalContext type annotation for jinja2.filters.do_join: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation EvalContext for jinja2.filters.do_join. Import of EvalContext failed: name 'EvalContext' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.sync_do_first: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.sync_do_first. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_first: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_first. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_last: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_last. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.do_random: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.do_random. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing EvalContext type annotation for jinja2.filters.do_urlize: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation EvalContext for jinja2.filters.do_urlize. Import of EvalContext failed: name 'EvalContext' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_truncate: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_truncate. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_wordwrap: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_wordwrap. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing t.Union[str, HasHTML] type annotation for jinja2.filters.do_striptags: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation t.Union[str, HasHTML] for jinja2.filters.do_striptags. Import of HasHTML failed: name 'HasHTML' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.Literal[\"common\", \"ceil\", \"floor\"] type annotation for jinja2.filters.do_round: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.Literal[\"common\", \"ceil\", \"floor\"] for jinja2.filters.do_round. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.sync_do_groupby: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.sync_do_groupby. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_groupby: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_groupby. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.sync_do_sum: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.sync_do_sum. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_sum: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_sum. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.filters.do_attr: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.filters.do_attr. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.sync_do_map: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.sync_do_map. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.do_map: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.do_map. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.sync_do_select: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.sync_do_select. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.do_select: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.do_select. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.sync_do_reject: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.sync_do_reject. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.do_reject: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.do_reject. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.sync_do_selectattr: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.sync_do_selectattr. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.do_selectattr: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.do_selectattr. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.sync_do_rejectattr: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.sync_do_rejectattr. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.do_rejectattr: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.do_rejectattr. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing EvalContext type annotation for jinja2.filters.do_tojson: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation EvalContext for jinja2.filters.do_tojson. Import of EvalContext failed: name 'EvalContext' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.prepare_map: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.prepare_map. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.prepare_select_or_reject: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.prepare_select_or_reject. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.select_or_reject: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.select_or_reject. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Context type annotation for jinja2.filters.async_select_or_reject: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Context for jinja2.filters.async_select_or_reject. Import of Context failed: name 'Context' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing 'te.Deque[Token]' type annotation for jinja2.lexer.TokenStream._pushed: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation 'te.Deque[Token]' for jinja2.lexer.TokenStream._pushed. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.lexer.Lexer.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.lexer.Lexer.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.lexer.compile_rules: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.lexer.compile_rules. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.lexer.get_lexer: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.lexer.get_lexer. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.nativetypes.NativeCodeGenerator.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.nativetypes.NativeCodeGenerator.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.Literal['\\n', '\\r\\n', '\\r'] type annotation for jinja2.nativetypes.NativeEnvironment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.Literal['\\n', '\\r\\n', '\\r'] for jinja2.nativetypes.NativeEnvironment.__init__. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] type annotation for jinja2.nativetypes.NativeEnvironment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Sequence[typing.Union[str, typing.Type[ForwardRef('Extension')]]] for jinja2.nativetypes.NativeEnvironment.__init__. Import of Extension failed: name 'Extension' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('BaseLoader')] type annotation for jinja2.nativetypes.NativeEnvironment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('BaseLoader')] for jinja2.nativetypes.NativeEnvironment.__init__. Import of BaseLoader failed: name 'BaseLoader' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('BytecodeCache')] type annotation for jinja2.nativetypes.NativeEnvironment.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('BytecodeCache')] for jinja2.nativetypes.NativeEnvironment.__init__. Import of BytecodeCache failed: name 'BytecodeCache' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.nodes.EvalContext.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.nodes.EvalContext.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('Environment')] type annotation for jinja2.nodes.Node.environment: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('Environment')] for jinja2.nodes.Node.environment. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.nodes.Node.set_environment: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.nodes.Node.set_environment. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing t.Optional[Environment] type annotation for jinja2.nodes.Const.from_untrusted: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation t.Optional[Environment] for jinja2.nodes.Const.from_untrusted. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.parser.Parser.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.parser.Parser.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.NoReturn type annotation for jinja2.parser.Parser.fail: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.NoReturn for jinja2.parser.Parser.fail. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.NoReturn type annotation for jinja2.parser.Parser.fail_unknown_tag: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.NoReturn for jinja2.parser.Parser.fail_unknown_tag. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing te.NoReturn type annotation for jinja2.parser.Parser.fail_eof: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation te.NoReturn for jinja2.parser.Parser.fail_eof. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing 'Environment' type annotation for jinja2.runtime.Context.environment: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation 'Environment' for jinja2.runtime.Context.environment. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.runtime.Context.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.runtime.Context.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.runtime.Macro.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.runtime.Macro.__init__. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing Environment type annotation for jinja2.runtime.new_context: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation Environment for jinja2.runtime.new_context. Import of Environment failed: name 'Environment' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing typing.Optional[ForwardRef('LoopRenderFunc')] type annotation for jinja2.runtime.LoopContext.__init__: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation typing.Optional[ForwardRef('LoopRenderFunc')] for jinja2.runtime.LoopContext.__init__. Import of LoopRenderFunc failed: name 'LoopRenderFunc' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing 'te.Deque[t.Any]' type annotation for jinja2.utils.LRUCache._queue: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation 'te.Deque[t.Any]' for jinja2.utils.LRUCache._queue. Import of te failed: name 'te' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}, {"message": ["Failed to run TYPE_CHECKING code while parsing t.Optional[VisitCallable] type annotation for jinja2.visitor.NodeVisitor.get_visitor: No module named 'typing_extensions'"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 132}, {"message": ["Error parsing type annotation t.Optional[VisitCallable] for jinja2.visitor.NodeVisitor.get_visitor. Import of VisitCallable failed: name 'VisitCallable' is not defined"], "category": "UserWarning", "filename": "/home/runner/.cache/pypoetry/virtualenvs/jinja2-pdoc-bjdabDNH-py3.10/lib/python3.10/site-packages/pdoc/doc_types.py", "lineno": 148}]